auspice-client/customisations/languageSelector.js

```diff
@@ -1,16 +1,50 @@
 import React from "react"; // eslint-disable-line
 import ISO6391 from "iso-639-1/build/index";
 
-/* This function is duplicated from ../../src/utils but included here to
-(a) keep server & client code seperate and (b) avoid any transpiling errors
-during client building */
-const parseNarrativeLanguage = (narrative) => {
-  const urlParts = narrative.split("/");
+/* This and some of the following functions are reused
```

Comment:
(Notes from a conversation Eli & I had today) -- This solution is fine for now. As auspice's extension machinery improves (there are some promising PRs at the moment) then these functions can be lifted to a more central place within the repo.